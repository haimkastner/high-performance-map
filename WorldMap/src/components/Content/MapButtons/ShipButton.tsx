import React from 'react';
import { MapButtonIcon } from './MapButtonIcon';

export class ShipButton extends React.Component {
  public render() {
    return <MapButtonIcon imageName={'movement-path-4333-shape-1@2x.png'}></MapButtonIcon>;
  }
}
