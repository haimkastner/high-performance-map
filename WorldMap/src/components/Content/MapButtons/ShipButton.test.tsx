import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import { ShipButton } from './ShipButton';

describe('ShipButton Component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<ShipButton />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('matches the snapshot', () => {
    const tree = renderer.create(<ShipButton />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  // tslint:disable-next-line: no-empty
  describe('ShipButton Component Logic', () => {});
});
