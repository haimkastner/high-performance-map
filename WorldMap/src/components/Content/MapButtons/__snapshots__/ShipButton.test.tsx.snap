// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShipButton Component matches the snapshot 1`] = `
<div
  className="MuiBox-root MuiBox-root-4"
>
  <img
    height={25}
    src="movement-path-4333-shape-1@2x.png"
    width={25}
  />
</div>
`;
